docs-v2/pages/workflows/settings.mdx

```diff
@@ -112,6 +112,14 @@ By default, workflows run with {process.env.MEMORY_LIMIT} of memory. If you're p
 
 Increasing your workflow's memory gives you a proportional increase in CPU, so increasing your workflow's memory can reduce its overall runtime and make it more performant.
 
+<Callout type="info">
+**Reducing Workflow Latency**
+
+To reduce latency from the trigger to the end of your workflow, consider increasing the memory allocation to at least 512MB. This will result in significantly better performance compared to the default 256MB.
```

Comment:
To avoid duplicating the content you have in the troubleshooting guide, maybe just simplify and say:

> See [our guide on running workflows faster](/troubleshooting#how-can-my-workflow-run-faster)